std/assembly/math.ts

```diff
@@ -1519,19 +1518,22 @@ export namespace NativeMathf {
     var sign_ = <i32>(hx >> 31);
     hx &= 0x7FFFFFFF;
     if (hx >= 0x42AEAC50) {
-      if (hx >= 0x42B17218 && !sign_) {
-        x *= Ox1p127f;
-        return x;
+      if (hx >= 0x42B17218) {
+        if (!sign_) {
+          x *= Ox1p127f;
+          return x;
+        } else {
+          if (hx >= 0x42CFF1B5) return 0;
+        }
       }
-      if (sign_ && hx >= 0x42CFF1B5) return 0;
     }
     var hi: f32, lo: f32;
     var k: i32;
     if (hx > 0x3EB17218) {
       if (hx > 0x3F851592) {
         k = <i32>(invln2 * x + builtin_copysign<f32>(0.5, x));
       } else {
-        k = 1 - sign_ - sign_;
+        k = 1 - (sign_ << 1);
```

Comment:
Does this improve something?

Reply:
I think yes because `sign_ - sign_` is two reads from locals, `sign_ << 1` only one read operation